### ISSUE TITLE
App exits to home screen when replaying a movie

### ISSUE THREAD

**Issue by HunterGReynolds** [jellyfin-archive/jellyfin-roku-legacy#489]
**Describe the bug**
When wantching a movie, if you allow it to run until the end and try to hit play again on the same movie, the app will exit.

**To Reproduce**

1. Play any movie
2. Allow the movie to play until it stops on its own
3. Click play once again
4. Be at the roku home screen

**Expected behavior**
The movie plays again.

**Comment by neilsb** [comment]
I can't reproduce this with the current channel store version (v 1.4.11).

If you can still reproduce this, can you confirm if you're going into the movie detail screen first and playing the movie from there?  Does it then return you to the Movie Detail screen once playback is finished, and you just press "OK" to play the movie again, and this is when it crashes?

**Comment by neilsb** [comment]
I believe this was resolved by the introduction of the SceneManager, and further protection is in place with PR #585.

Closing this issue as it doesn't seem to be an issue any more, but if it still happening then please raise a new issue.
